docker/1.4.1/py2/Dockerfile.eia

```diff
@@ -1,21 +1,50 @@
-FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py2.7
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python
+ARG PIP=pip
+ARG HEALTH_CHECK_VERSION=1.3.2
```

Comment:
does this need to be updated to 1.3.3. I see in TF dockerfiles
https://github.com/aws/sagemaker-tensorflow-serving-container/pull/77/files#diff-579c54887ad8272b572b526bc4d10f5eR10

Reply:
Updated to 1.3.3.

Thanks.